src/main/java/core/db/DbUtil.java

```diff
@@ -5,10 +5,16 @@
 
 public class DbUtil {
 
-    static public Integer getNullableInt(ResultSet rs, String cl) throws SQLException {
+    public static Integer getNullableInt(ResultSet rs, String cl) throws SQLException {
         Integer res = rs.getInt(cl);
         if (!rs.wasNull()) return  res;
         return null;
     }
 
+    public static Boolean getNullableBoolean(ResultSet rs, String cl) throws SQLException {
```

Comment:
usually it is good practice to put the @nullabale operator to faciilitate the work of the debugger

Reply:
public static @Nullable Integer getNullableInt(ResultSet rs, String cl) throws SQLException

@akasolace  do you mean it this way? i will change that

Reply:
yes, and you will need to import org.jetbrains.annotations.Nullable;